src/components/TeachersPage/TeacherPopUp/TeacherPopUp.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
```

Comment:
```suggestion
import { FC } from 'react'
```
change everywhere,pls
i'll do it in my own components too